src/upload/form/mod.rs

```diff
@@ -19,6 +19,7 @@ use project_assets::ProjectAssets;
 use text_blob::TextBlob;
 use wasm_module::WasmModule;
 
+// these likely belong elsewhere, perhaps in a mod for project type specific behavior.
```

Comment:
nit: i think we should remove this comment and either file an issue or fix it up in this refactor, thoughts?

Reply:
i'm happy to do it in this refactor, i'll put it on my Friday list

Reply:
jk we need this in, can make an issue.

Reply:
https://github.com/cloudflare/wrangler/issues/1083